packages/cli/src/services/project-checker.ts

```diff
@@ -111,6 +111,9 @@ class LowerCliVersionError extends Error {
 }
 
 async function compareVersionsAndDisplayMessages(cliVersion: string, projectVersion: string): Promise<void> {
+  // If were running tests, the version is *. Also, its a good escape hatch in case the user uses other versioning
+  if (cliVersion === '*' || projectVersion === '*') return
```

Comment:
I'd expand the comment to explain how this works and why, I'm not sure if I understand it.